runbotics/runbotics-orchestrator-ui/src/src-app/components/Tile/ProcessTile/ProcessTileTagList/ProcessTileTagList.tsx

```diff
@@ -22,13 +22,14 @@ const TAGS_CONTAINER_MARGIN_VALUE = 67;
 const TAG_RIGHT_MARGIN_VALUE = 8;
 
 const ProcessTileTagList: FC<ProcessTileTagListProps> = ({
-    tags, searchValue
+    tags, searchValue, refProcessTileContent
 }) => {
     const { translate } = useTranslations();
 
     const [isTagBoxExpanded, setIsTagBoxExpanded] = useState<boolean>(false);
     const [isTagListMultiLine, setIsTagListMultiLine] = useState<boolean>(true);
     const [tagsWidth, setTagsWidth] = useState<number>(0);
+    const [expandedHeight, setExpandedHeight] = useState<number>(0);
```

Comment:
initial 0 can cause UI flicking

Reply:
or is it only when you're expanding tags view?

Reply:
So should I leave it undefined?

Reply:
Yes, it is only used when tagBox is expanded.